### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by karmaester**
- Prepare the directory structure
- Implement the App component
- Implement the BooksList component
- Implement the Book component
- Implement the BooksForm component
- Define the Redux actions and reducers
- Initialize the Redux store

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import './App.css';
+import BookList from './components/BookList';
```
Comment (whiz25) [inline src/App.js]: - Place this component in the `src/components` directory where it rightly belongs

File: src/components/BookList.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BookList = ({ books }) => (
+  <div className="container">
+    <table className="table table-striped">
```
Comment (whiz25) [inline src/components/BookList.js]: - Move this to `src/containers`

File: src/components/BookForm.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+
+const cat = [
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
```
Comment (whiz25) [inline src/components/BookForm.js]: - Move this component to `src/containers` folder

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this project. You managed to meet most of the requirements for this milestone.
### Highlights 🌵 🌵 🌵 
- `App`, `BookForm`, `BookList` components and reducers were implemented as required.

### Change(s) to make 🙅🏿 🙅🏿 🙅🏿 
- Update readme file with the necessary information for cloning, installing packages and running the app locally. Refer to this [template](https://github.com/microverseinc/readme-template)
- Check my comments for suggestions


### Martin Afani
#### Slack: whiz25

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi Developers, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**